Non-RPC/mumble-ping.py

```diff
@@ -15,10 +15,7 @@
 	fmt = sys.argv[3]
 else:
 	fmt = "Version $v, $u/$m Users, $p, $b"
-if '-v' in sys.argv:
-	verbose = True
-else:
-	verbose = False
+verbose = '-v' in sys.argv
```

Comment:
For handling command-line arguments, the `argparse` module should be used (imo)

Reply:
Good point. I thought about this too but did not want to introduce additional dependencies. But everything is working as intended as is or am I missing something?

Reply:
> I thought about this too but did not want to introduce additional dependencies.

`argparse` is actually not a dependency. It is part of the core Python library and is thus always available without any further installation requirements :point_up: 
(At least afaik)

> But everything is working as intended as is or am I missing something?

Idk. I did not test these changes nor do I have the necessary insights to actually assess whether the code should be doing what it is supposed to do :shrug:

Reply:
If this PR is about porting the script to python 3 I think it is better to leave introducing unrelated stuff out. The script did not use argparse before either. This keeps it smaller/simpler, which is better for reviewing and the merge commit description and later reading it.

It would definitely be nice to replace manual handling with argparse afterwards though.

Reply:
Fair enough :+1:

Reply:
So... are we ready to merge?